test/e2e/.eslintrc.js

```diff
@@ -4,11 +4,10 @@ module.exports = {
 		'../../eslint/config.js',
 	],
 	env: {
-		mocha: true,
+		jest: true
```

Comment:
Since this extends our base configuration, we shouldn't need to specify `env` at all now I think.

Reply:
I tried but it didn't work :)